stable/prometheus/templates/node-exporter-role.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if and .Values.nodeExporter.enabled .Values.rbac.create }}
-{{- if .Values.nodeExporter.podSecurityPolicy.enabled }}
+{{- if .Values.podSecurityPolicy.enabled }}
```

Comment:
what if we did `{{- if or (default .Values.nodeExporter.podSecurityPolicy.enabled false) (.Values.podSecurityPolicy.enabled) }}` for these checks, and then put a deprecation message in the NOTES.txt ? 

that way its not yet a breaking change, but is also preparing people for deprecation of the old value ?

Reply:
or if we want the breaking change we could error if `.Values.nodeExporter.podSecurityPolicy.enabled` is set with a error message with upgrade instructions.

Reply:
done!